### PR TITLE
Use websocket for notifications

### LINKED ISSUE #2127: Use WebSockets for notifications
Now that we only support Qt >= version 5.5, see https://github.com/GNS3/gns3-gui/issues/2080, we can use WebSockets for notifications.

### PR DESCRIPTION
**Pull request by noplay**
Fix #2127

### CODE REVIEW

**Comment by grossmj** [comment]
When overwriting an existing project:

```
****** Exception detected, traceback information saved in /home/grossmj/.config/GNS3/exceptions.log ******
PLEASE REPORT ON https://www.gns3.com

Traceback (most recent call last):
  File "/home/grossmj/PycharmProjects/gns3-gui/gns3/qt/__init__.py", line 247, in partial
    return func(*args, **kwargs)
  File "/home/grossmj/PycharmProjects/gns3-gui/gns3/http_client.py", line 563, in _readyReadySlot
    callback(answer, server=server, context=context)
TypeError: _event_received() got an unexpected keyword argument 'server'
```

**Comment by noplay** [comment]
Fixed it was due to old Qt version. But you will not be able to test websockets if you don't use Qt 5.6 or later

File: gns3/main.py [hunk 1]
```diff
@@ -184,8 +184,8 @@ def exceptionHook(exception, value, tb):
     if sys.version_info < (3, 4):
         raise SystemExit("Python 3.4 or higher is required")
 
-    if parse_version(QtCore.QT_VERSION_STR) < parse_version("5.0.0"):
-        raise SystemExit("Requirement is PyQt5 version 5.0.0 or higher, got version {}".format(QtCore.QT_VERSION_STR))
+    if parse_version(QtCore.QT_VERSION_STR) < parse_version("5.5.0"):
```
Comment (grossmj) [inline gns3/main.py]: Should we still required Qt version 5.5 or above?

File: gns3/qt/__init__.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
 import logging
 log = logging.getLogger("qt/__init__.py")
 
-from PyQt5 import QtCore, QtGui, QtNetwork, QtWidgets, Qt
+from PyQt5 import QtCore, QtGui, QtNetwork, QtWidgets, QtWebSockets, Qt
```
Comment (grossmj) [inline gns3/qt/__init__.py]: QtWebSocket is imported below and shouldn't be imported here since it is not installed by default.

Comment (noplay) [inline gns3/qt/__init__.py]: Cleaned

File: gns3/main.py (hunk 1 above)
Comment (noplay) [inline gns3/main.py]: I think we will no longer test with version before 5.5 and we suspect they are a source of issues.  All current LTS distribution provide Qt 5.5 or later.

Comment (grossmj) [inline gns3/main.py]: Ok, let's keep >= 5.5 then.

**Pull request merged**